gamestonk_terminal/economy/alphavantage_view.py

```diff
@@ -49,10 +55,6 @@ def realtime_performance_sector(raw: bool, export: str):
         # TODO: convert to mpl
         df_rtp.plot(kind="bar")
```

Comment:
can you check if this thing returns an `ax`?
like `ax = df_rtp.plot(kind="bar")`
like this you will be able to add the external axes part and do the styling with all the helpers